spec/transformers/api/v3/user_transformer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Api::V3::UserTransformer do
+  describe 'to_response' do
+    let!(:user) { create(:user) }
+
+    subject(:response) { described_class.to_response(user) }
+
+    before do
+      allow(Api::V3::Transformer).to receive(:to_response)
+        .with(user)
+        .and_return({ 'user' => 'attributes'})
```

Comment:
Add sensitive attributes to this stubbed response to make the assertion below meaningful.